src/output-flex.cpp

```diff
@@ -1185,6 +1264,7 @@ void output_flex_t::way_delete(osmid_t osm_id)
 
 void output_flex_t::relation_delete(osmid_t osm_id)
 {
+    mark_relation_members(osm_id);
```

Comment:
Do we call the output_delete before deleting from middle? I suspect not.

Reply:
I believe we do.